packages/airnode-deployer/docker/Dockerfile

```diff
@@ -15,9 +15,10 @@ COPY --from=api3/airnode-artifacts /build/packages/airnode-deployer/dist ${appDi
 COPY packages/airnode-deployer/docker/entrypoint.sh /entrypoint.sh
 
     # Install external dependencies
-RUN apk add --update --no-cache su-exec git && \
+RUN export ARCH=$([ $(arch) == "x86_64" ] && echo "amd64" || echo "arm64") && \
```

Comment:
Won't this echo both `amd64` and `arm64` when arch is x86_64?

Reply:
You're right, I somehow missed this, I will rectify it now. I must have only tested the previous version when I ran the arm64 deployer build.

Reply:
Fixed, sorry about that.